src/main/resources/application.properties

```diff
@@ -1 +1,20 @@
+server.port=8080
```

Comment:
no need to define it here?

Reply:
sometimes the port 8080 is occupied

Reply:
Where should it be defined instead?

Reply:
@JannesVolkens just leave it here for now.